github/orgs_teams.go

```diff
@@ -37,10 +40,23 @@ type Team struct {
 	RepositoriesURL *string       `json:"repositories_url,omitempty"`
 }
 
+// Invitation represents a team member's inviation status
+type Invitation struct {
+	ID        *int       `json:"id,omitempty"`
+	Login     *string    `json:"login,omitempty"`
+	Email     *string    `json:"email,omitempty"`
+	Role      *string    `json:"role,omitempty"`
+	CreatedAt *time.Time `json:"created_at,omitempty"`
+}
+
 func (t Team) String() string {
 	return Stringify(t)
 }
 
+func (i Invitation) String() string {
+	return Stringify(i)
+}
```

Comment:
I think it'll be more readable and consistent to keep the type and its `String` method together. For example, see [here](https://github.com/google/go-github/blob/b272bb85ab53b610b5660988998feb0e0294bd2b/github/admin.go#L17-L60).